src/js/components/modals/AppModalComponent.jsx

```diff
@@ -235,6 +240,10 @@ var AppModalComponent = React.createClass({
           size="md"
           onDestroy={this.props.onDestroy}>
         <form method="post" role="form" onSubmit={this.handleSubmit}>
+          <button onClick={event => event.preventDefault()}
+              className="enter-trap">
+            Enter trap
+          </button>
```

Comment:
What do you think of getting rid of `.enter-trap` and doing this:

```
<button onClick={event => event.preventDefault()} style={{display: "none"}} />
```

We don't need that class and the text node is also unnecessary.